src/IcuEC/IcuTranslitAutoConfigDialog.cs

```diff
@@ -22,23 +22,45 @@ public class IcuTranslitAutoConfigDialog : SilEncConverters40.AutoConfigDialog
 		[DllImport("IcuTranslitEC.dll", EntryPoint = "IcuTranslitEC_ConverterNameList_start", CallingConvention = CallingConvention.Cdecl)]
         static extern unsafe int CppConverterNameList_start();
 
-#if __MonoCS__
-		[DllImport("IcuTranslitEC.dll", EntryPoint = "IcuTranslitEC_ConverterNameList_next", CallingConvention = CallingConvention.Cdecl)]
-        [return: MarshalAs(UnmanagedType.LPStr)]
-        static extern unsafe string CppConverterNameList_next();
+        static string CppConverterNameList_next()
+        {
+            if (Util.IsUnix)
+            {
+                return CppConverterNameList_next_Linux();
+            }
+            else
+            {
+                return CppConverterNameList_next_Windows();
+            }
+        }
+
+        static string CppGetDisplayName(string strID)
+        {
+            if (Util.IsUnix)
+            { return CppGetDisplayName_Linux(strID); }
+            else
+            {
+                return CppGetDisplayName_Windows(strID);
+            }
+        }
+
+        [DllImport("IcuTranslitEC.dll", EntryPoint = "IcuTranslitEC_ConverterNameList_next", CallingConvention = CallingConvention.Cdecl)]
+		[return : MarshalAs(UnmanagedType.LPStr)]

```

Comment:
Some indent problems here, check the whole file.

Reply:
Done